docs: adds streaming messages section to sending message page (#1384)
slackapi/bolt-python: docs/english/concepts/message-sending.md

            blocks=blocks,
            text="Pick a date for me to remind you"
        )
```

## Streaming messages {#streaming-messages}

You can have your app's messages stream in to replicate conventional AI chatbot behavior. This is done through three Web API methods:

* [`chat_startStream`](/reference/methods/chat.startstream)
* [`chat_appendStream`](/reference/methods/chat.appendstream)
* [`chat_stopStream`](/reference/methods/chat.stopstream)

> The Python Slack SDK provides a [`chat_stream()`](https://docss.slack.dev/tools/python-slack-sdk/reference/web/client.html#slack_sdk.web.client.WebClient.chat_stream) helper utility to streamline calling these methods. Here's an excerpt from our [Assistant template app](https://github.com/slack-samples/bolt-python-assistant-template/blob/main/listeners/assistant/assistant.py)

Reviewer: ```suggestion
The Python Slack SDK provides a [`chat_stream()`](/tools/python-slack-sdk/reference/web/client.html#slack_sdk.web.client.WebClient.chat_stream) helper utility to streamline calling these methods. Here's an excerpt from our [Assistant template app](https://github.com/slack-samples/bolt-python-assistant-template)
```

🐍 suggestion: Removing perhaps more ambiguous links for the exploring readers!

Reviewer: 🗣️ note: I understand the linked file of the sample exists on a branch at the moment, but I'm not certain that we will keep this file structure into the future after this lands. Hoping to avoid updates later with this!

```python
streamer = client.chat_stream(
    channel=channel_id,
    recipient_team_id=team_id,
    recipient_user_id=user_id,
    thread_ts=thread_ts,
)

> for event in returned_message:

Reviewer: ```suggestion
# Loop over OpenAI response stream
# https://platform.openai.com/docs/api-reference/responses/create
for event in returned_message:
```

🤖 suggestion: Feel free to change or ignore this, but it might be useful to note where the `returned_message` is from or that it's a streaming response?

    if event.type == "response.output_text.delta":
        streamer.append(markdown_text=f"{event.delta}")
    else:
        continue

feedback_block = create_feedback_block()
streamer.stop(blocks=feedback_block)
```

> In that example, a [feedback buttons](/reference/block-kit/block-elements/feedback-buttons-element) block element is passed to `streamer.stop` — this provides feedback buttons to the user at the bottom of the message.

Reviewer: ```suggestion
In that example, a [feedback buttons](/reference/block-kit/block-elements/feedback-buttons-element) block element is passed to `streamer.stop` to provide feedback buttons to the user at the bottom of the message. Interaction with these buttons will send a block action event to your app to receive the feedback.
```

💌 suggestion: Perhaps a kind note on how to make use of this block?
